python/kserve/test/test_server.py

```diff
@@ -146,13 +145,7 @@ def _parserequest(self, request):
         return record1
 
     def preprocess(self, request, headers: Dict[str, str] = None):
-        if isinstance(request, CloudEvent):
```

Comment:
I think the cloudevent attributes are passed as headers can we assert that instead ?

Reply:
done